src/main/java/org/fundacionjala/pivotal/workspace/EditWorkspace.java

```diff
@@ -0,0 +1,146 @@
+
+    @FindBy(how = How.ID, using = "confirm_delete")
+    private WebElement buttonConfirmDeleteWorkspace;
+
+    /**
+     * Method for get the last workspace created.
+     * @return lastWorspaceCreatedLink.
+     */
+    public WebElement getLastWorkspaceCreatedLink() {
+        return CommonMethods.waitWebElement(lastWorkspaceCreatedLink);
+
+    }
+
+
+    /**
+     * Method for get the last workspace created.
+     * @return the workspace name input.
+     */
+    private WebElement getTxtWorkspaceName() {
+        return CommonMethods.waitWebElement(txtWorkspaceName);
+
+    }
+
+    /**
+     * Method for get the last workspace created.
+     * @return the workspace name input.
+     */
+    private WebElement getButtonSaveWorkspace() {
+        return CommonMethods.waitWebElement(buttonSaveWorkspace);
+
+    }
+
+    /**
+     * Method for get the last workspace created.
+     * @return the workspace name input.
+     */
+    private WebElement getButtonDeleteWorkspace() {
+        return CommonMethods.waitWebElement(buttonDeleteWorkspace);
+
+    }
+    /**
+     * Method for get the last workspace created.
+     * @return the workspace name input.
+     */
+    private WebElement getButtonConfirmDeleteWorkspace() {
+        return CommonMethods.waitWebElement(buttonConfirmDeleteWorkspace);
+
+    }
+    /**
+     * Method for get the last workspace created.
+     * @return the workspace name input.
+     */
+    public WebElement getSuccessMessage() {
+        return CommonMethods.waitWebElement(successMessage);
+
+    }
+
+    /**
+     * Method for set Workspace name.
+     * @param workspaceName String.
+     */
+    private void setWorkspaceName(final String workspaceName) {
+        CommonMethods.setWebElement(getTxtWorkspaceName(), workspaceName);
+    }
+
+    /**
+     * Method for press click over the last workspace created.
+     */
+    private void clickLastWorkspaceCreatedLink() {
+        CommonMethods.clickWebElement(this.getLastWorkspaceCreatedLink());
+    }
+
+
+
+    /**
+     * Method for press click buttton for save a new workspace.
+     */
+    private void clickButtonSaveWorkspace() {
+        CommonMethods.clickWebElement(this.getButtonSaveWorkspace());
+    }
+
+    /**
+     * Method for press click buttton for save a new workspace.
+     */
+    private void clickButtonDeleteWorkspace() {
+        CommonMethods.clickWebElement(this.getButtonDeleteWorkspace());
+    }
+
+    /**
+     * Method for press click buttton for save a new workspace.
+     */
+    private void clickButtonConfirmDeleteWorkspace() {
+        CommonMethods.clickWebElement(this.getButtonConfirmDeleteWorkspace());
+    }
+
+    /**
+     * Method to Create workspace .
+     * @param workspaceName String.
+     */
+    public void editWorkspace(final String workspaceName) {
+
+        CommonMethods.clickWebElementWithJavascriptExecutor(getLastWorkspaceCreatedLink());
+
+        //   this.clickLastWorkspaceCreatedLink();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)